rdr_service/dao/bq_questionnaire_dao.py

```diff
@@ -192,23 +192,29 @@ def make_bqrecord(self, p_id, module_id, latest=False, convert_to_enum=False):
                         'questionnaire_id',
                         'external_id',
                         'status',
-                        'status_id',
-                        'test_participant'
+                        'status_id'
                     ):
                         continue
 
                     fld_value = getattr(bqr, fld_name, None)
-                    if fld_value is None:  # Let empty strings pass.
+                    # Check to see if the field has been forced to boolean and set the default.
+                    if field['type'] == BQFieldTypeEnum.INTEGER.name:
+                        setattr(bqr, fld_name, 0)
+                    if fld_value is None:
                         continue
                     # question responses values need to be coerced to a String type.
                     if isinstance(fld_value, (datetime.date, datetime.datetime)):
                         setattr(bqr, fld_name, fld_value.isoformat())
-                    else:
-                        setattr(bqr, fld_name, str(fld_value))
-
+                    # Check to see if the field has been forced to boolean, set to 1 because we have a value.
```

Comment:
The over-commenter in me is wondering if it might be helpful to expand on this comment, i.e., that the only other BQFieldTypeEnum.INTEGER fields in a module schema are in the "ignore list" (`if fld_name in...`) above.

Reply:
Yes, all questionnaire module fields are forced to String because answers may be multiple data types.  IE: A free text can have a string answer or maybe be a code answer of 'PMI_Skip', so it's easiest to force all answers to String.